mne/io/_read_raw.py

```diff
@@ -57,13 +59,23 @@ def _read_unsupported(fname, **kwargs):
 suggested = {
     ".vmrk": dict(brainvision=partial(_read_unsupported, suggest=".vhdr")),
     ".amrk": dict(brainvision=partial(_read_unsupported, suggest=".ahdr")),
-    ".eeg": dict(brainvision=partial(_read_unsupported, suggest=".vhdr")),
 }
 
 # all known file formats
 readers = {**supported, **suggested}
 
 
+def split_name_ext(fname):
+    """Return name and supported file extension."""
+    maxsuffixes = max([ext.count(".") for ext in supported])
+    suffixes = Path(fname).suffixes
+    for i in range(-maxsuffixes, 0):
+        ext = "".join(suffixes[i:]).lower()
```

Comment:
for readability we try to avoid one-letter iterators
```suggestion
    for si in range(-maxsuffixes, 0):
        ext = "".join(suffixes[si:]).lower()
```